src/main/java/frc/robot/subsystems/TurretSubsystem.java

```diff
@@ -49,63 +50,63 @@
 
 public class TurretSubsystem extends SubsystemBase {
 
-  private CANSparkMax turretMotor;
+  private final CANSparkMax turretMotor;
   private SingleJointedArmSim turretSim;
```

Comment:
Not a required change, but it is a good practice to separate final attributes from non-final attributes.

Reply:
Also: this could probably be final, as well.